auth_from_http_remote_user/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Author: Laurent Mignon
+# Copyright 2014-2018 'ACSONE SA/NV'
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+{
+    'name': 'Authenticate via HTTP Remote User',
+    'version': '11.0.1.0.0',
+    'category': 'Tools',
+    'author': "Acsone SA/NV,Odoo Community Association (OCA)",
+    'maintainer': 'ACSONE SA/NV',
+    'website': 'https://github.com/OCA/server-auth',
+    'depends': ['base', 'web', 'base_setup'],
+    "license": "AGPL-3",
+    "installable": True,
```

Comment:
you can drop these flags

Reply:
All three last ones  (installable, auto_install and application) ?

Reply:
Well that is what I did. I am curious where I can find the default values for those in the code ?

Reply:
Default values are here https://github.com/odoo/odoo/blob/0050019390daa759ac5882ded79a6ee50ab4749e/odoo/modules/module.py#L310
So indeed they can all go.